### PR TITLE
closes #103 | retrieve order details

### LINKED ISSUE #103: Customers can view details about orders created by them

### PR DESCRIPTION
**Pull request by adetola-ralph**
*Description of Task*
- Retrieve information about a single order

### CODE REVIEW

**Comment by bolorundurowb** [comment]
@adetola-ralph Are you done with this feature?

**Comment by adetola-ralph** [comment]
@bolorundurowb yes I am, abeg merge this one and #108 abeg. I am tired of waiting

File: tests/orders.test.js [hunk 1]
```diff
@@ -289,15 +294,113 @@ describe('Orders', () => {
...
+            fittingDate: '11-03-2017',
+            deliveryDate: new Date(),
+            tailorId
+          })
+          .end((err, res) => {
+            orderId = res.body.data._id;
+            done();
+          });
+        });
+
```
Comment (bolorundurowb) [inline tests/orders.test.js]: Hey man, you didnt put the `allows` describe

File: tests/orders.test.js [hunk 2]
```diff
@@ -289,15 +294,113 @@ describe('Orders', () => {
...
     });
   });
 
   describe('Server error', () => {
     before(() => {
-      const err = new Error();
-      serverErrorStub = sinon.stub(Order, 'create').callsFake(({ phone, password }, cb) => {
-        cb(err);
-      });
+      serverErrorStub = sinon.stub(Order, 'create')
```
Comment (bolorundurowb) [inline tests/orders.test.js]: You dont have to assign this to a variable, the stub works without the assignment

**Pull request merged**